CollAction/Services/Project/ProjectService.cs

```diff
@@ -285,62 +249,107 @@ private string GetParticipantCsvLine(ApplicationUser user)
         private string EscapeCsv(string str)
             => $"\"{str?.Replace("\"", "\"\"")}\"";
 
+        public async Task<FindProjectsViewModel> FindProject(int projectId)
+        {
+            var project =  await _context
+                .Projects
+                .Include(p => p.Category)
+                .Include(p => p.Location)
+                .Include(p => p.BannerImage)
+                .Include(p => p.ParticipantCounts)
+                .FirstOrDefaultAsync(p => p.Id == projectId);
+
+            return project != null
+                ? CreateFindProjectsViewModel(project)
+                : null;
+        }
+
+        public IQueryable<FindProjectsViewModel> FindProjects(Expression<Func<Models.Project, bool>> filter, int? limit, int? start)
```

Comment:
This doesn't seem to include the "start" parameter, will break pagination

Reply:
Good catch! Missed that one during the merge.

Reply:
Good catch! Missed that while merging.